style.css

```diff
@@ -695,3 +719,165 @@ contact me section
     width: 159px;
   }
 }
+
+/* The Modal (background) */
+.modal {
+  display: none; /* Hidden by default */
+  position: fixed; /* Stay in place */
+  z-index: 1; /* Sit on top */
+  padding-top: 100px; /* Location of the box */
+  left: 0;
+  top: 0;
+  width: 100%; /* Full width */
+  height: 100%; /* Full height */
+  overflow: auto; /* Enable scroll if needed */
+  background-color: rgb(0, 0, 0); /* Fallback color */
+  background-color: rgba(0, 0, 0, 0.4); /* Black w/ opacity */
+}
+
+/* Modal Content */
+.modal-content {
+  background-color: #fefefe;
+  margin: auto;
+  padding: 20px;
+  border: 1px solid #888;
+  width: 920px;
+}
```

Comment:
- [ ] KIndly make your modal content a bit rounded at the corners by adding a border-radius, also add a margin-bottom to it to make it look like the Figma design
your design :point_down: 
![Screenshot from 2022-12-22 10-22-22](https://user-images.githubusercontent.com/66097371/209102442-a8b706cf-a33d-4a24-9985-1ed9f37f5403.png)